javascript/javascript1/week4/js1week4.js

```diff
@@ -0,0 +1,133 @@
+// voice assistant
+function getReply(commandRequested) {
+    const index = getCommandIndex(commandRequested);
+
+    switch (index) {
+        case 0: { greeting(commandRequested); break; }
+        case 1: { sayMyName(); break; }
+        case 2: { getToday(); break; }
+        case 3: { addTodo(commandRequested); break; }
+        case 4: { removeTodo(commandRequested); break; }
+        case 5: { setTimer(commandRequested); break; }
+        default: {
+            console.error('Unkown command .. please check again !');
+            return;
+        }
+    }
+}
+
+
+function greeting(commandRequested) {
+    commandRequested = commandRequested || "";
+    const splittedCommand = commandRequested.split(' ');
+    if (splittedCommand.length >= 4) {
+        //['my,name,is,maha,{karzoun}]
+        //[maha,{karzoun}]
+        const clientName = splittedCommand.splice(3, splittedCommand.length).join(' ');
+
+        if (client.name) {
+            console.log(`welcone back ${client.name} :D
+ `);
+        } else {
+            console.log(`welcone ${clientName} , Thank you for using my program :)`);
+            client.name = clientName;
+        }
+    }
+}
+
+function sayMyName() {
+    if (!client.name) {
+        console.log('You are not registerd yet ! ');
+        return;
+    }
+    console.log(`You are ${client.name}`);
+}
+
+function getToday() {
+    const date = new Date();
+    console.log(`Today is ${date.getDay()} of ${months[date.getMonth()]}`);
```

Comment:
`getDay()` gives you a day of the week. For the date, it's `getDate()`. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/getDate

Reply:
i did the change thanks ^^